lib/mappings.js

```diff
@@ -0,0 +1,28 @@
+exports.impression = [
+	["list", "nm"]
+];
+
+exports.action = [
+	["type", "pa"],
+	["id", "ti"],
+	["list", "pal"],
+	["affiliation", "ta"],
+	["revenue", "tr"],
+	["tax", "tt"],
+	["shipping", "ts"],
+	["coupon", "tcc"],
+	["step", "cos"],
+	["option", "col"]
+];
+
+exports.product = [
+	["id", "id"],
+	["name", "nm"],
+	["brand", "br"],
+	["category", "ca"],
+	["variant", "va"],
+	["price", "pr"],
+	["quantity", "qt"],
+	["coupon", "cc"],
+	["position", "ps"]
+];
```

Comment:
Because I didn't feel like looping through `Object.keys()` and I didn't want to introduce `lodash` as a dependency.